readme.md

```diff
@@ -1,12 +1,12 @@
 # Mocha generator [![Build Status](https://secure.travis-ci.org/yeoman/generator-mocha.svg?branch=master)](http://travis-ci.org/yeoman/generator-mocha)
 
-> Yeoman generator for [Mocha](http://visionmedia.github.io/mocha/)
+> Yeoman generator for [Mocha](http://mochajs.org/)
 
 
 ## Install
 
 ```
-$ npm install --global generator-mocha
+$ npm install -g generator-mocha
```

Comment:
Having this written in the long form is something we indented, we even explicitly changed it: https://github.com/yeoman/generator-mocha/commit/17665469adf0f08ff92e0c67a81ed4988351d002. Is there any strong reason you want us to change it?